ansible/roles/ocp4-workload-iot-managed-user/templates/user_project.yaml.j2

```diff
@@ -0,0 +1,944 @@
+  generation: 2
+  labels:
+    app: iot-data-generator
+    app.kubernetes.io/component: iot-data-generator
+    app.kubernetes.io/instance: iot-data-generator
+    app.kubernetes.io/part-of: iot-data-generator
+  name: iot-data-generator
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - annotations:
+      openshift.io/generated-by: OpenShiftWebConsole
+      openshift.io/imported-from: quay.io/evanshortiss/rhtr-2020-iot-datagen
+    from:
+      kind: DockerImage
+      name: quay.io/evanshortiss/rhtr-2020-iot-datagen
+    generation: 2
+    importPolicy: {}
+    name: latest
+    referencePolicy:
+      type: Source
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    openshift.io/generated-by: OpenShiftWebConsole
+  labels:
+    app: iot-data-generator
+    app.kubernetes.io/component: iot-data-generator
+    app.kubernetes.io/instance: iot-data-generator
+    app.kubernetes.io/name: ""
+    app.kubernetes.io/part-of: iot-data-generator
+    app.openshift.io/runtime: ""
+    app.openshift.io/runtime-version: latest
+  name: iot-data-generator
+spec:
+  ports:
+  - name: 8080-tcp
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: iot-data-generator
+    deploymentconfig: iot-data-generator
+  sessionAffinity: None
+  type: ClusterIP
+status:
+  loadBalancer: {}
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    openshift.io/generated-by: OpenShiftWebConsole
+  creationTimestamp: "2020-09-11T12:02:35Z"
+  labels:
+    app: iot-sse-server
+    app.openshift.io/runtime: quarkus
+    app.kubernetes.io/component: iot-sse-server
+    app.kubernetes.io/instance: iot-sse-server
+    app.kubernetes.io/part-of: iot-sse-server
+  name: iot-sse-server
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: iot-sse-server
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        openshift.io/generated-by: OpenShiftWebConsole
+      creationTimestamp: null
+      labels:
+        app: iot-sse-server
+        deploymentconfig: iot-sse-server
+    spec:
+      containers:
+      - image: quay.io/evanshortiss/rhtr-2020-quarkus-sse
+        imagePullPolicy: Always
+        name: iot-sse-server
+        ports:
+        - containerPort: 8081
+          protocol: TCP
+        resources: {}
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      securityContext: {}
+      terminationGracePeriodSeconds: 30
+
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  annotations:
+    openshift.io/image.dockerRepositoryCheck: "2020-09-11T12:02:35Z"
+  creationTimestamp: "2020-09-11T12:02:34Z"
+  generation: 2
```

Comment:
`generation`, `creationTimestamp` and `openshift.io/image.dockerRepositoryCheck` annotation should not be in version control.